encoders/image/CLIPImageEncoder/__init__.py

```diff
@@ -10,29 +10,27 @@
 
 class CLIPImageEncoder(BaseTorchEncoder):
     """
-    Encode data from a `np.ndarray` of shape `BatchSize x (Channel x Height x Width)` into
-    a `np.ndarray` of shape `Batchsize x EmbeddingDim`.
+    Encodes data from a `np.ndarray` of shape `BatchSize x (Channel x Height x Width)` into
+    a `np.ndarray` of shape `Batchsize x EmbeddingDimension`.
 
-    Internally, :class:`CLIPImageEncoder` wraps the `CLIP` model
-    https://github.com/openai/CLIP
-
-    :param model_name: the name of the model. Supported models include ``ViT-B/32`` and ``RN50``.
- 
+    Internally, :class:`CLIPImageEncoder` wraps the `CLIP` modeL from https://github.com/openai/CLIP
     """
     def __init__(self, model_name: str ='ViT-B/32',
                  *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.model_name = model_name
 
     def post_init(self):
+        """
+        :param model_name: the name of the model. Supported models include ``ViT-B/32`` and ``RN50``.
```

Comment:
there is no model_name param here, the param list of constructor go at the top of the class